### DISCUSSION TITLE
The doc tab in "just --chose" does not work with modules

### DISCUSSION THREAD

**Discussion by oivviors** [casey/just#2133] [Q&A]
I have a folder tree with justfiles in three levels.
The documentation tab in "just --choose" will work fine for recipes in the first level, but not for the recipes in sub modules.

I'm able to run these nested recipes but their docs do not work.

Any ideas for solutions?

<img width="750" alt="Screenshot 2024-06-05 at 21 26 59" src="https://github.com/casey/just/assets/142509931/c9557d35-f661-4ccf-8d3c-1fcda2b0084d">

**Comment by casey** [comment]
You are in luck! I just released 1.28, which should fix this. Give it a shot and let me know if it works.

**Reply by oivviors** [reply]
Wonderful tool BTW.

**Reply by oivviors** [reply]
Just tried it out. The double colons between names is replaced by a space but it still doesn't work.

**Reply by casey** [reply]
Thanks for the kind words! Whoops, this is definitely still not working. I reopened #2110 which tracks this.

**Reply by oivviors** [reply]
Fixed in 1.29 I see. Very nice.

**Reply by oivviors** [reply]
Another thing, that I'd love to see is an option to edit the selected command invokation before running it. That way I could tweak parameters before running.
